actionview/test/template/form_helper_test.rb

```diff
@@ -3240,6 +3268,12 @@ def test_fields_for
     assert_dom_equal expected, output_buffer
   end
 
+  def test_fields_for_field_id
+    fields_for(:post, @post) do |fields|
+      assert_equal "post_title", fields.field_id(:title)
```

Comment:
What would be the expected value before this change?

Reply:
In this case, `post_title` is the expected value, so the behavior does not change. This commit adds missing test coverage for that behavior, so that there aren't regressions.